### DISCUSSION TITLE
Pydantic decorators?

### DISCUSSION THREAD

**Discussion by letto4135** [fastapi/fastapi#3863] [Questions] [answered]
I don't know enough to be able to do it, but wouldn't it be much cooler and easier if we could combine the schemas for fastapi and the models for pydantic into a single class with just a decorator to say either hey this is a pydantic model, figure it out from the typing, or hey this is a fastapi schema, figure it out from the column types.
Maybe this already exists and I just don't know enough? Feels like it would be a very useful enhancement if not.

**Comment by letto4135** [comment]
Just kidding apparently! tiangolo is already making pydantic-sqlalchemy. Gorgeous

**Comment by Kludex** [comment, marked as answer]
https://sqlmodel.tiangolo.com/
